presentation/common/src/presentation-common/EC.ts

```diff
@@ -236,6 +241,16 @@ export namespace RelatedClassInfo {
     };
   }
 
+  /** Deserialize compressed [[RelatedClassInfo]] from JSON */
+  export function fromCompressedRelatedClassInfoJSON(compressedInfoJSON: RelatedClassInfoJSON<string>, classesMap: { [id: string]: CompressedClassInfoJSON }): RelatedClassInfoJSON {
```

Comment:
The function is already under `RelatedClassInfo` namespace - no need to repeat that in function name